Packs/CortexXDR/Classifiers/classifier-PaloAltoNetworks_CortexXDR.json

```diff
@@ -1,219 +1,17 @@
 {
-	"brandName": "Cortex XDR - IR",
+	"fromVersion": "6.0.0",
```

Comment:
Do we want to add here defaultType ?

Reply:
@aviadl If we support now default type in the classifiers, then yes.

Reply:
@anara123 its `defaultIncidentType` - we had it before